armis_sdk/core/armis_client.py

```diff
@@ -87,18 +87,38 @@ def __init__(
         except ValueError:
             self._default_backoff = 0
 
+    def _get_proxy_config(self):
+        """Get proxy configuration from environment variables."""
+        proxy = (
+            os.getenv("HTTPS_PROXY")
+            or os.getenv("https_proxy")
+            or os.getenv("HTTP_PROXY")
+            or os.getenv("http_proxy")
+        )
+        return proxy
```

Comment:
```suggestion
        return os.getenv("HTTPS_PROXY") or os.getenv("HTTP_PROXY")
```

though TBH the whole thing can be inlined in the caller